src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -50,7 +53,8 @@ final class ENATaskScheduler {
 	typealias CompletionHandler = (() -> Void)
 
 	private func registerTasks() {
-		registerTask(with: .primaryBackgroundTask, taskHander: executeBackgroundTask(_:))
+		registerTask(with: .primaryBackgroundTask, taskHander: executeExposureDetectionRequest(_:))
```

Comment:
`taskHander` ? shouldn't it be `taskHandler`?

Reply:
Yep, have fixed that now!